tests/test_csc.py

```diff
@@ -146,7 +146,7 @@ def test_lifecycle_behavior(self):
                 # Telemetry loop should no longer be running
                 self.assertTrue(harness.csc.telemetry_loop_task.done())
 
-        asyncio.get_event_loop().run_until_complete(doit())
+        asyncio.run(doit())
```

Comment:
Feel free to say this is out of scope for this ticket, but please consider using asynctest. It makes the tests look much nicer and seems to improve feedback when a test fails. It is now part of T&S's standard packages. The basics:
```
import asynctest

class MyTestCase(asyncTest.TestCase):
    async def test_some_asynchronous_think(self):
         # your asynchronous test code here
```
Thus you can skip the `async def doit()...asyncio.run(doit())` nonsense.

Note:
* Test methods can be synchronous or asynchronous.
* setUp and tearDown can be synchronous or asynchronous.

Also this allows you to easily ditch the `Harness` test class, if you have one:
* If Harness constructor takes no arguments then use `async def setUp(self)` to build the CSC and Remote and wait for them to start and `async def tearDown(self)` to close them and wait for them to close.
* If Harness constructor has arguments then I initialize `self.csc` and `self.remote` (or whatever attributes Harness had) to `None`, then add an asynchronous method `make_csc` that takes the necessary arguments, sets the attributes and waits for them to start. Then close them in `tearDown` and wait, if not None.

See ts_watcher for an example. I haven't gotten rid of all the Harness classes in ts_salobj yet.

Reply:
Done.